samples/run_jobs.py

```diff
@@ -35,22 +35,30 @@
 name = f'Samples (Scheduled by GitHub workflow: {datetime_now})'
 new_study = NewJob(owner, project, recipe, name=name, client=api_client)
 
+# get all unique artifacts
+artifacts = set()
+for sample_run in sample_runs:
+    for value in sample_run.values():
+        input_path = Path(__file__).parent.resolve().joinpath(value)
+        if input_path.exists():
```

Comment:
Shouldn't the file always exist? Otherwise, the test will fail. We better fail here before submitting the studies.

Reply:
I get what you mean. What I am really checking there is if the value is a path, because the value can also be a non-path recipe input in which case we don't want to upload it.

Then I propose we separate the artifact inputs (everything that needs an upload) from the other inputs in the JSON file. I added additional inputs in the example below, but "inputs" can also be an empty dictionary if we don't want to change any non-artifact inputs from the default (which is probably true in most cases).
```
[
    {
        "artifacts": {
            "model": "artifacts/sample_model_grid.hbjson",
            "wea": "artifacts/Boston-Logan Intl AP_0_8759.wea"
        },
        "inputs": {
            "north": "90"
        }
    },
    {
        "artifacts": {
            "model": "artifacts/sample_model_grid_aperture_groups.hbjson",
            "wea": "artifacts/Boston-Logan Intl AP_0_8759.wea"
        },
        "inputs": {
            "radiance-parameters": "-ab 5 -ad 10000"
        }
    }
]
```

Reply:
I like the proposal. This is how they are actually set in Argo too. They are called `artifacts` vs `parameters`.